src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNWeight.java

```diff
@@ -123,5 +123,13 @@ public Scorer scorer(LeafReaderContext context) throws IOException {
     public boolean isCacheable(LeafReaderContext context) {
         return true;
     }
+
+    public static float score(float score) {
+        if (score >= 0) {
+            return 1 / (1 + score);
+        } else {
+            return 2 + 1 / (score - 1);
```

Comment:
private static float normalizeScore(float score) {
    if (score >= 0)
         return 1 / ( 1 + score );
   return 2 + ( 1 / (score - 1) );

Reply:
Thanks for making changes. 
1. Do we need this to be public? 
2. Shall we add parenthesis around formula? For better readability, i am thinking we don't want reader to remember the precedence logic to understand the formula. 
What do you think? @chenqi0805

Reply:
> 1. Do we need this to be public?
Sorry, I missed the access modifier in your previous comment. The reason I maintained it public is to reuse it in `KNNScoringSpace`. If made private, we will need to duplicate the logic there. Let me know what you think.

> 2. Shall we add parenthesis around formula? For better readability, i am thinking we don't want reader to remember the precedence logic to understand the formula.
What do you think? @chenqi0805

Sure. Will add the parenthesis.